tmk_core/protocol/usb_hid/override_wiring.c

```diff
@@ -2,7 +2,7 @@
  * To keep Timer0 for common/timer.c override arduino/wiring.c.
  */
 #define __DELAY_BACKWARD_COMPATIBLE__
-#include <util/delay.h>
+#include <wait.h>
```

Comment:
```suggestion
#include "wait.h"
```
And use `wait_ms`/`_us` below